pages/docs/integrations/grouparoo-mixpanel.mdx

```diff
@@ -0,0 +1,160 @@
+---
+title: "Mixpanel"
+date: "2022-02-04"
+pullQuote: "Learn how to work with the Mixpanel Integration as a Destination."
+pageNavSelector: "h2"
+---
+
+<PluginDocsLinks plugin="Mixpanel" />
+
+## Overview
+
+Mixpanel is a marketing analysis tool that helps you understand how users interact with your products. Grouparoo can help you sync data from a centralized warehouse to Mixpanel for enhanced data quality and assurance in the most up to date information for your Mixpanel analysis.
+
+### Destinations
+
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>Destination Name</td>
+      <td>Destination Description</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>
+          <a href="#mixpanel-export-profiles">Mixpanel Export Profiles</a>
+        </strong>
+      </td>
+      <td>Export Records to Mixpanel as User Profiles</td>
+    </tr>
+  </tablebody>
+</table>
+
+## Setup
+
+You will need a Mixpanel account and some additional information from your MixPanel account in order to sync data to Mixpanel.
+
+Navigate to Settings and choose the project you want to connect to from the Subheading called Project Settings. On the Overview Page, you can find the Project Id.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/mixpanel/setup/setup-1.png"
+  alt="Mixpanel Setup"
+  width={750}
+  height={395}
+/>
+<br />
+
+Two other values you will need to configure a Mixpanel App in Grouparoo are found under Project Settings → Service Accounts. There, you will see a list of usernames. The Secret associated with each username is only available upon creation. If you need to create a new service account to access a secret for configuration, be sure to set the project role for that account to Admin.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/mixpanel/setup/setup-2.png"
+  alt="Mixpanel Setup"
+  width={750}
+  height={405}
+/>
+<br />
+
+## App
+
+To connect Mixpanel to Grouparoo, you [create an App](https://www.grouparoo.com/docs/config/apps) which tells Grouparoo the details of how to connect to Mixpanel. This specifies the account used for any Destinations.
+
+### App Options
+
+In Grouparoo’s UI config, you specify the options to connect to Mailjet and test the connection.
+
+<ImageInBrowserFrame
+  src="/images/docs/integrations/mixpanel/app/options.png"
+  alt="Mixpanel App Options"
+  width={750}
+  height={426}
+/>
+<br />
+
+<table className="docTable">
+  <tablebody>
+    <tr>
+      <td>Option Name</td>
+      <td>Description</td>
+    </tr>
+    <tr>
+      <td>
+        <strong>Token</strong> (required)
+      </td>
+      <td>Mixpanel API Token</td>
```

Comment:
We explained how to find the Project ID and user credentials.  Should we mention where to find the API Token, too?